### DISCUSSION TITLE
Widget idea

### DISCUSSION THREAD

**Discussion by loftwah** [deleteman/the-hive#3] [Ideas]
G'day mate, I hope you are doing well. I had an idea earlier that could work with this project. We could create a browser extension that works as a widget (possibly can do PWA) and those in the community could have a shared 'notes' tab where things could be shared between the group like gotchas, tips, actual accurate and reasonably private reviews so they're honest, voting, alternatives etc. It could get annoying with day-to-day browsing, but if we encouraged people to use a different browser profile to use it, it could be a commonly shared tool among everyone in the community.

Find a really good deal, like a coupon or something? Share it with the group. Have a bad experience? Let the group know what happened. I got the idea from the Daily Dev widget, so if you want an idea of what I'm thinking could look like, that would be pretty close.

**Comment by deleteman** [comment]
Oh I love the idea of a companion widget like that! 
Definitely v2 material since it would need some well defined APIs to back it up, but I'm writing this down for the future. I love it.

**Reply by loftwah** [reply]
Definitely, I'm thinking long-term. I was actually considering putting something together that is very similar to what you have, so depending on the direction I might be able to contribute here instead.

**Reply by deleteman** [reply]
Well, if you want to read a high-level overview of where I want to ideally take this project, you can read this post: https://deleteman123.medium.com/the-open-source-industry-has-a-big-problem-and-here-is-how-i-want-help-fix-it-8f18e095ec73

Everything could change based on feedback of course, but I'd like to turn this project into a real source of income for OSS developers... eventually.

**Reply by loftwah** [reply]
Nice, I'll have a read. I am not a greedy person and have no expectations besides being involved with Open Source, anything else is a bonus IMO,
